service/templates/cloudformation/main.yaml

```diff
@@ -3,5 +3,6 @@ Description: Main CloudFormation stack.
 Resources:
   {{template "launch_configuration" .}}
   {{template "autoscaling_group" .}}
+  {{template "worker_policy" .}}
```

Comment:
I'm assuming Cloud Formation doesn't care about the ordering here?

But having the worker policy before the launch config makes more sense to me. Since it's needed by the launch config.

Reply:
Makes sense, done